config/environments/production.rb

```diff
@@ -33,12 +33,12 @@
 
   # Enable serving of images, stylesheets, and javascripts from an asset server
   # config.action_controller.asset_host = "http://assets.example.com"
-
   config.action_mailer.delivery_method = :sendmail
   config.action_mailer.raise_delivery_errors = true
   config.action_mailer.default_options = {
     from: 'no-reply@opensnp.org'
   }
+  config.action_mailer.asset_host = "https://opensnp.org"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.